include/exiv2/bmpimage.hpp

```diff
@@ -26,15 +26,6 @@ namespace Exiv2 {
  */
 class EXIV2API BmpImage : public Image {
  public:
-  ~BmpImage() override = default;
-  //! @name NOT Implemented
-  //@{
-  //! Copy constructor
-  BmpImage(const BmpImage&) = delete;
```

Comment:
I think we need to be more careful with these changes. Even in `0.27` we were "not implementing" the copy constructor & assignment in purpose, with the aim of making some classes not copyable.  This would be an important API change, and probably we do not want to make such objects copyable anyways. 

The deletion of the `default` destructors looks fine to me though

Reply:
for all the removals here, the base class has deleted copy constructor and operator= which is inherited by the derived classes. All of them are not copyable unless such a thing is implemented per class.

Reply:
Fair enough, I did not notice the `Image` class already was deleting the copy constructor and operator.